### DISCUSSION TITLE
Non Responsive App Owners

### DISCUSSION THREAD

**Discussion by nana283** [community/community#31647] [Other Feature Feedback, Questions, & Ideas]
I want to ask/complain about the non responsive attitude of the App Owners.  We have been asking about an app that is not functioning properly and receiving no response.  Not even an acknowledgement.

Is this a standard accepted policy?  Does hosting on Github not have any restrictions on that?  Cause it is very frustrating.

https://github.com/stankovski/quran-phone/issues/235

Thank you

**Comment by queenofcorgis** [comment]
If you're asking how to use/interact with a piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by nana283** [comment]
I, and a few others, did all you suggested.

That wasn't my Q.

My Q was after we did all your suggestions, those owners/maintainers do not
reply!

Does GitHub have a policy that they should?

Thank you
Inas

**Comment by jfullstackdev** [comment]
last update was 2018

![image](https://user-images.githubusercontent.com/47092464/190449751-739338da-29fc-40c1-b65d-f303169dffdb.png)

so I think it's clear it's not active anymore

plus, there is no specific license, the README did not even mentioned that license, the rule of GitHub is, without explicitly telling it, then it's still copyright and cannot be distributed 

as for the non-responsiveness, because it's open source without license and not even be allowed to be distributed, then you cannot ask for support if they don't want to

usually, an open source project that is being maintained can have that kind of support but it will be paid, particularly if the client wants some modification from that open source project, this is where they can make money out of open source

and this is also the reason why there are still companies providing custom software for clients because it's full support because they are paying for it, here in the Philippines, I'm currently employed in one of those companies

**Comment by nana283** [comment]
Thank you for the clarification.

I appreciate the feedback and information.  I understand that GitHub can't
control the managers, but shouldn't it have a say on "non-active"
applications left on the site and confuse users :/

Thank you again for responding and the feedback
Inas
Thank you

Inas A Taha
*ITSM Strategy Alignment and Architect SME*
*646.884.1806*

**Comment by airtower-luna** [comment]
What kind of "say" should they have? If they make *any* kind of change to users' repositories outside violations of ToS or applicable law they'll *very* quickly lose users' (that also means customers') trust. If I couldn't rely on my repositories staying where and as I put them, why would I put them here? And if others' repositories would disappear or get changed at will, why would I look here? That would be unacceptable and self-destructive behavior for a code hosting platform, and that's before considering the harm of lost public knowledge. Even abandoned code might teach someone something, and (where the license allows) someone might pick up development again.

**Comment by nana283** [comment]
There are probably more than one option for governance ....

Simple answer to very elaborated and articulated reply ... Service
Management is about the "USER" ....if as a user I can't trust that this a
repository for functional, managed and governed applications .. be sure it
is not one I will *trust* to look for what they have to offer ...or be
willing to recommend it to anyone else.

But I guess you have your priorities.

Thank you
